pkgs/development/libraries/science/math/scalapack/default.nix

```diff
@@ -12,6 +12,8 @@ stdenv.mkDerivation rec {
     sha256 = "0p1r61ss1fq0bs8ynnx7xq4wwsdvs32ljvwjnx6yxr8gd6pawx0c";
   };
 
+  patches = [ ./openmpi4.patch ];
```

Comment:
Can you add a comment with a link to this patch being submitted upstream or an explanation for why this patch is specific to nixpkgs?

Reply:
scalapack is not very actively developed. I will see if can get submit this patch upstream. The patch is self written.

Reply:
Maybe to clarify on that question: according to repology there is only linuxbrew which has openmpi-4 and scalapack packaged but I do not see a solution to the problem there. We are probably the first distro to package both and have the problem solved.